supabase/functions/manage-livekit-room/index.ts

```
@@ -0,0 +1,126 @@
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${generateManagementToken(apiKey, apiSecret)}`,
+          },
+          body: JSON.stringify({
+            name: roomName,
+            emptyTimeout: emptyTimeout || 300, // デフォルト5分
+            maxParticipants: maxParticipants || 100,
+          }),
+        });
+
+        if (!createResponse.ok) {
+          const error = await createResponse.text();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)